### PR TITLE
fix/subflow-tasks-attempts

### LINKED ISSUE #2098: Subflow task logs are not shown
### Expected Behavior

_No response_

### Actual Behaviour

See https://demo.kestra.io/ui/executions/io.kestra.bmu/call-simple-subflow/2euoLKmgwfLH0lAlHu9uU/logs where there is no logs boxes due to Subflow tasks not producing any attempt making it a edge case

### Steps To Reproduce

_No response_

### Environment Information

- Kestra Version: 0.12.0-SNAPSHOT
- Operating System (OS / Docker / Kubernetes):
- Java Version (If not docker):

### Example flow

```
id: call-simple-subflow
namespace: io.kestra.bmu
tasks:
  - id: subflow
    type: io.kestra.core.tasks.flows.Flow
    flowId: simple-subflow
    namespace: io.kestra.bmu
```

```
id: simple-subflow
namespace: io.kestra.bmu
tasks:
  - id: hello
    type: io.kestra.core.tasks.log.Log
    message: Kestra team wishes you a great day! 👋
```

### PR DESCRIPTION
**Pull request by brian-mulier-p**
closes #2098

### CODE REVIEW

**Review by loicmathieu — COMMENTED** [review]
this seems legit but can you explain why it is needed now and works without before?

**Comment by brian-mulier-p** [comment]
I explained it quickly on the issue but attempts are not really used (or at least in this case subflow fail almost never so you don't need to replay the task itself) except for logs display and not having attempt on "Flow" tasks makes it an edge case

File: core/src/main/java/io/kestra/core/runners/ExecutorService.java [hunk 1]
```diff
@@ -598,7 +599,9 @@ private Executor handleFlowTask(final Executor executor) {
                     }
                 } catch (Exception e) {
                     workerTaskResults.add(WorkerTaskResult.builder()
-                        .taskRun(workerTask.getTaskRun().withState(State.Type.FAILED))
+                        .taskRun(workerTask.getTaskRun().withState(State.Type.FAILED)
+                            .withAttempts(Collections.singletonList(TaskRunAttempt.builder().state(new State().withState(State.Type.FAILED)).build()))
```
Comment (Skraye) [inline core/src/main/java/io/kestra/core/runners/ExecutorService.java:603]: Would be more readable if expand on multiple line

**Review by Skraye — APPROVED** [review]
LGTM
Just long one line are not very readable

**Pull request merged**